### ISSUE TITLE
Tabs have too little contrast

### ISSUE THREAD

**Issue by rlucke** [opencast/opencast-admin-interface#1271]
![Image](https://github.com/user-attachments/assets/6d1508de-2709-46e8-8fd1-fdd7602891c6)

![Image](https://github.com/user-attachments/assets/430285c8-4f65-430f-8c28-1f5ae8ec96dc)

In the Event details dialogue, the tabs in the second row have too little contrast at 2.57:1. Only the active tab has an acceptable contrast of 4.6:1. However, a value of more than 7:1 should be aimed for.

**Comment by rlucke** [comment]
Accessibility is regulated by the European Accessibility Act (EAA) and in Germany by the Barrierefreiheitsstärkungsgesetz (BFSG). Contrast ratios are therefore not a ‘nice to have’. If they are missing, this is a mistake that needs to be rectified.

**Comment by oas777** [comment]
Thanks for taking part in this bug bash, @rlucke. Please keep in mind this is organised to gather issues we can reasonably solve. Which requires prioritisation and @michiHB is in charge of that. If you feel strongly about accessibility feel free to contact me to discuss funding options.
